net-analyzer/openvas-scanner/files/openvas.conf

```diff
@@ -1 +1 @@
-db_address = /tmp/redis.sock
+db_address = /run/redis/redis.sock
```

Comment:
This either requires a new file name or that we immediately drop openvas-scanner-21.4.4.-r2. Otherwise, the -r2 ebuild would install a different file depending on the time of installation.

Reply:
It was wrong even for 21.4.
https://wiki.gentoo.org/wiki/Greenbone_Vulnerability_Management#Redis

Reply:
Yes, but really the /etc/redis/redis.conf file inserted by dev-db/redis-7.0.11 has Unix socket lines commented.
The wiki says hot to modify it, even if in the wiki page there is a wrong reference to /etc/redis.conf  https://wiki.gentoo.org/wiki/Greenbone_Vulnerability_Management#Redis

If I undertand, you think it is better to use openvas-scanner's ebuild to add a 
an openvas-redis.conf
a  openvas-redis.service for systemd
a  openvas-redis.confd for openrc
to run a separate redis instance for gvm

I will try this approach to write a new service to start a redis instance dedicated to openvas.
However, I think it is better to leave the modification in openvas.conf for 21.4.4-r3 and to drop 21.4.4-r2 (it doesn't build).

Reply:
Hi @Flowdalic I have made some changes to use a separate instance of redis for openvas. Now I am making some tests before commit.
I have a question: do you think it would be good to create a systemd service to start all GVM service at once (notus-scanner, ospd-openvas, gvmd and gsad)?

Reply:
Hi @Flowdalic, I dropped 21.4.4.-r2, provided a separate redis instance configuration and added a gvm systemd service.
Can you resolve this conversation?